lib/results.js

```diff
@@ -4,6 +4,7 @@ var through = require('through');
 var resumer = require('resumer');
 var inspect = require('object-inspect');
 var hasOwn = Object.prototype.hasOwnProperty;
+var invalidYaml = RegExp.prototype.test.bind(/\:|\-|\?/);
```

Comment:
Using `.bind` breaks ES3 compatibility. Can you use the `function-bind` module here instead?

Alternatively, don't use bind, capture `RegExp#test` in a closure, and make a wrapper function that `.call`s it - but that's less robust against `Function#call` being broken.